src/composables/transactionData.ts

```diff
@@ -141,7 +141,7 @@ export function useTransactionData({
 
   const isDex = computed((): boolean => isTxDex(innerTx.value, dexContracts.value));
 
-  const isDexAllowance = computed((): boolean => (
+  const isAllowance = computed((): boolean => (
```

Comment:
If it's not related to dex I suggest to add some comment about what "allowance" actually is in this context.